certbot/main.py

```diff
@@ -790,6 +790,13 @@ def install(config, plugins):
     except errors.PluginSelectionError as e:
         return str(e)
 
+    custom_cert = (config.key_path and config.cert_path)
```

Comment:
Should also change line 805 to match

Reply:
The current implementation on line 805 catches both of the cases - user providing both `--key-path` and `--cert-path` and configuration object values being populated from either interactively selected or user provided `--cert-name`.

Are you suggesting something in line with `if custom_cert or config.certname:` ?

Reply:
I meant this as a nit -- you already create a variable that captures this boolean, so might as well use `custom_cert` on line 805 in place of `config.key_path and config.cert_path`

Reply:
Ah, right. Maybe I should elaborate a bit; these values get populated from `config.certname` obtained from CLI or by interactive selection in line 804 to normalize the handling from there on, so the initial variable value might not reflect the actual state at this point.